### DISCUSSION TITLE
Problem with tests hanging after they start to run (using puppeteer helper)

### DISCUSSION THREAD

**Discussion by jayudey-wf** [codeceptjs/CodeceptJS#3576] [Q&A]
I'm wondering if people have seen where tests appear to be hanging after 'starting' the test.

Here is an example of what I see in the console output when running in debug mode before the process stops reporting anything

```
2023-02-03T19:47:31.032317730Z   Add markup from toolbar
2023-02-03T19:47:31.032654348Z  › Test Timeout: 300s

Failure Reason:
```

On a successful run I see logging like this

```
2023-02-03T19:44:48.154071543Z   Add markup from toolbar
2023-02-03T19:44:48.154315772Z  › Test Timeout: 300s
2023-02-03T19:44:48.381567785Z  › [Dir] /testing/test/codeceptjs
2023-02-03T19:44:48.381668181Z >>Current Test FileName: file
```

My reporter options look like this

```
  mocha: {
    reporterOptions: {
      "codeceptjs-cli-reporter": {
        "stdout": "-",
        "options": {
          "verbose": true,
          "steps": false,
        }
      },
      "mocha-junit-reporter": {
          "stdout": "./output/console.log",
          "options": {
          "mochaFile": "./output/reports/junit-results.[hash].xml",
        }
      }
    },
  }
```

I'm wondering if there is some other course of action I could do to try and gather some more information about what is 'halting' the process so I can continue my investigation (currently this is happening ~1 in 30 runs)?

**Comment by adudyak** [comment]
1. try to turn off `verbose` flag
2. try to increase `--max-old-space-size` for NodeJS https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes
